sled-agent/src/bootstrap/early_networking.rs

```diff
@@ -672,7 +690,8 @@ impl<'a> EarlyNetworkSetup<'a> {
         &self,
         port_config: &PortConfig,
     ) -> Result<(PortSettings, PortId), EarlyNetworkSetupError> {
-        info!(self.log, "Building Port Configuration");
+        let log = self.log.new(o!("stage" => SWITCH_ZONE_INIT_STAGE));
```

Comment:
Do we need to create another logger here or could we pass in the one from `init_switch_config`? (If that one doesn't call us but some other method calls both it and us, maybe that method should create the logger and pass it down to both instead?)

Reply:
fixed, thanks!